packages/db-migrations/databases/fxa/patches/patch-138-139.sql

```diff
@@ -0,0 +1,27 @@
+-- Create carts table to store checkout Cart state when creating a subscription.
+-- An FxA user can have more than one Cart record e.g. if they have more than
+-- one subscription where each subscription was created in a different session.
+-- A Cart record may not be associated with an FxA user e.g. if they started
+-- the new user checkout flow but did not get to the point where we create an
+-- account.
+
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('138');
+
+CREATE TABLE IF NOT EXISTS carts (
+    id BINARY(16) NOT NULL PRIMARY KEY,
+    uid BINARY(16),
+    FOREIGN KEY (uid) REFERENCES accounts(uid) ON DELETE CASCADE,
+    state ENUM('new', 'upgrade', 'processing', 'success', 'fail') NOT NULL,
+    errorReasonId VARCHAR(255),
+    offeringConfigId VARCHAR(255) NOT NULL,
+    `interval` ENUM('day', 'week', 'month', 'year') NOT NULL,
```

Comment:
`interval` is a reserved word per https://dev.mysql.com/doc/refman/8.0/en/keywords.html, so I have to enclose it in backticks.